vault/assets/configuration/spec.yaml

```
@@ -29,6 +29,12 @@ files:
         example: http://localhost:8200/v1
         pattern: \w+
         type: string
+    - name: collect_secondary_dr
+      description: |
+        Enable this option to collect from the OpenMetrics endpoint of the secondary in a Vault replication cluster.
```

Small nit. Reading this makes me believe that if they enable this, we will automatically connect to a secondary and pull in metrics, but that doesn't seem to be the case. The wording should ideally encapsulate that if we're connecting to a secondary, we'll still collect the metrics from the OM endpoint.

My best try (english isn't my first language 🥲 ): 
```suggestion
        Enable this option to collect metrics from the OpenMetrics endpoint if the connected instance is a secondary in a Vault replication cluster.
```